tests/nested/core/core-snap-refresh-on-core/task.yaml

```diff
@@ -5,7 +5,7 @@ details: |
     revision to a new one. It expects to find a new snap revision in the
     channel pointed by the NESTED_CORE_REFRESH_CHANNEL env var.
 
-systems: [ubuntu-16.04-64]
+systems: [ubuntu-18.04-64]
```

Comment:
this test is specific to the core snap, so it naturally will have different behavior on uc16 than uc18, so can we just run this one on both systems?